pipelines/integration-test.yml

```diff
@@ -40,3 +66,27 @@ jobs:
         path: pr
         context: {{github_status_context}}
         status: failure
+
+  - name: tag-releases
+    public: true
+    serial: true
+    plan:
+    - aggregate:
+      - get: release-repository
+        trigger: true
+      - get: resource-version
+        params:
+          bump: minor
+      - task: run-tests
+        file: pr/ci/integration.yml
```

Comment:
I get this error from the `run-tests` task:
```
unknown artifact source: pr
```

I think `pr` needs to be renamed to `release-repository` to match the resource name.